app/models/user.rb

```diff
@@ -21,9 +21,11 @@ class User < ApplicationRecord
   validates :password, presence: true, length:
     {minimum: Settings.validation.pass_min}, allow_nil: true
   has_secure_password
-  before_save{email.downcase!}
+  before_save :downcase_email
+  before_create :create_activation_digest
 
   scope :latest, ->{order(created_at: :desc)}
+  scope :activated_true, ->{whree(activated: true)}
```

Comment:
```suggestion
  scope :activated_true, ->{where activated: true}
``` code này chạy dc luôn